x-pack/legacy/plugins/code/server/utils/format_parser.ts

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
this parser might worth a simple unit test in case any unexpected changes in the future.

Reply:
comment to raise this up.